presto-orc/src/test/java/com/facebook/presto/orc/TestSelectiveOrcReader.java

```diff
@@ -290,6 +290,52 @@ public void testFloats()
                                 1, FloatRange.of(1.0f, false, true, 7.0f, false, true, true))));
     }
 
+    @Test
+    public void testOrdering()
+            throws Exception
+    {
+        // We can now check that filters are called in the order we expect.
+        // Watch out for:
+        // - If you efficiently filter out all positions in a page by filter x, then filter x + 1 won't even be called, making your test fail
+        // - Only works for bigintrange and doublerange right now
+
+        // Setup
+
+        Random random = new Random(0);
+
+        List<Integer> values1 = newArrayList(limit(cycle(ImmutableList.of(1, 3, 5, 7, 11)), 200));
```

Comment:
don't use variable names like a1, a2, a3...; I suggest to inline values* and *Filter like so:

```
    @Test
    public void testFilterOrder()
            throws Exception
    {
        Random random = new Random(0);

        tester.testRoundTripTypesWithOrder(ImmutableList.of(INTEGER, INTEGER),
                ImmutableList.of(newArrayList(limit(cycle(ImmutableList.of(1, 3, 5, 7, 11)), NUM_ROWS)), randomIntegers(NUM_ROWS, random)),
                toSubfieldFilters(
                        ImmutableMap.of(
                                0, BigintRange.of(1, 1, true),
                                1, BigintRange.of(2, 200, true))),
                ImmutableList.of(ImmutableList.of(0, 1)));

        tester.testRoundTripTypesWithOrder(ImmutableList.of(INTEGER, INTEGER),
                ImmutableList.of(newArrayList(limit(cycle(ImmutableList.of(1, 3, 5, 7, 11)), NUM_ROWS)), randomIntegers(NUM_ROWS, random)),
                toSubfieldFilters(
                        ImmutableMap.of(
                                0, BigintRange.of(100, 100, false),
                                1, BigintRange.of(2, 200, true))),
                ImmutableList.of(ImmutableList.of(0)));

        tester.testRoundTripTypesWithOrder(
                ImmutableList.of(INTEGER, INTEGER, DOUBLE, arrayType(INTEGER)),
                ImmutableList.of(newArrayList(limit(cycle(ImmutableList.of(1, 3, 5, 7, 11)), NUM_ROWS)), createList(NUM_ROWS, i -> random.nextInt(200)), doubleSequence(0, 0.1, NUM_ROWS), nCopies(NUM_ROWS, randomIntegers(5, random))),
                ImmutableList.of(
                        ImmutableMap.of(
                                0, toSubfieldFilter(BigintRange.of(1, 1, true)),
                                1, toSubfieldFilter(BigintRange.of(0, 200, true)),
                                2, toSubfieldFilter(DoubleRange.of(0, false, false, 0.1, false, false, true)),
                                3, toSubfieldFilter("c[1]", BigintRange.of(4, 4, false)))),
                ImmutableList.of(ImmutableList.of(0, 1, 2, 3)));
    }
```

Reply:
Okay. Later, let's discuss why this is a preferred style. I can see why normally it'd be the case, but here I found named variables to be helpful in reasoning about the test and debugging it.